resources/js/app.js

```diff
@@ -5,12 +5,12 @@
  * building robust, powerful web applications using React + Laravel.
  */
 
-require('./bootstrap');
+//require('./bootstrap'); // I don't think I want to attach anything to Window, maybe we will rethink this later --- Drake
```

Comment:
we should load axios to the window and let bootstrap.js automatically send the csrf token without too much fuss: 

https://github.com/deepditch/deep.web/blob/4d0cb35afb45dadaf858401b9c48269146ffc451/resources/js/bootstrap.js#L17-L39

Wouldn't hurt

Reply:
I'll put that back for now. However, I still think there is a better way to resolve axios than by attaching to window. Especially if we use some of the same services for the native app, attaching to window won't be an option.